metrics/ci_integrations/test/client/github_actions/test_utils/mock/github_actions_mock_server.dart

```diff
@@ -197,7 +197,9 @@ class GithubActionsMockServer extends ApiMockServer {
     final runsPerPage = _extractPerPage(request);
     final pageNumber = _extractPage(request);
 
-    List<WorkflowRun> workflowRuns = _generateWorkflowRuns(status);
+    List<WorkflowRun> workflowRuns = _generateWorkflowRuns(
+      status,
+    ).reversed.toList();
```

Comment:
I believe it would be better to update the `_generateWorkflowRuns` method to make it generate builds in the correct order instead of reversing it here.